cms/jinja2/templates/pages/information_page.html

```diff
@@ -0,0 +1,34 @@
+{% extends "templates/base_page.html" %}
+{% from "components/related-content/_macro.njk" import onsRelatedContent %}
+
+{% block main %}
+    <p>{{ page.summary }}</p>
+
+    {% if page.last_updated %}
+        <h5>Last Updated: {{ page.last_updated }}</h5>
+    {% endif %}
+
+    {% include_block page.content %}
+
+    {% if related_pages %}
+        {# fmt:off #}
+        {{-
+            onsRelatedContent({
+                "ariaLabel": _('Related content'),
+                "rows": [{
+                    "id": 'related-content',
+                    "title": _('Related content'),
+                    "itemsList": related_pages
+                }]
+            })
+        -}}
+        {# fmt:on #}
+    {% endif %}
+
+{% endblock %}
+{% if page.has_equations %}
```

Comment:
Duplicated code here. Now that the partial we need is in the `cms/jinja2/templates/base_page.html` file. Since this page is inheriting from `base_page.html` we don't need this code here.

FYI: In the case that the partial wasn't in the `base_page.html`, you would be importing the partial from `/cms/jinja2/templates/partials/content_scripts.html` and using it here, not rewriting the whole code manually here as that's why we have a partial.

Reply:
@zerolab Beat me to it!

Reply:
🫡 Done